CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Version 0.5.1-dev (changes since 0.5.1 go here)
 
+* Added ``skbio.io.format.embl`` for reading and writing EMBL files for ``DNA``, ``RNA``, ``Protein`` and ``Sequence`` classes.
```

Comment:
Yes. I will fix it. I wrote this line before deciding to concentrate myself on DNA